README.Rmd

```diff
@@ -59,12 +67,21 @@ square means estimates by using `emmeans`.
   * [Kenward-Roger](https://openpharma.github.io/mmrm/main/articles/kenward.html)
     adjusted degrees of freedom and coefficients covariance matrix.
   * [Coefficient Covariance](https://openpharma.github.io/mmrm/main/articles/coef_vcov.html)
-* Model inference:
-  * Supports REML and ML.
-  * Supports weights.
+</details>
+<details><summary>CPP Backend</summary>
+
 * Fast implementation using C++ and automatic differentiation to
-obtain precise gradient information for model fitting. See [here](https://openpharma.github.io/mmrm/main/articles/algorithm.html)
-for details of the model fitting algorithm used in `mmrm`.
+obtain precise gradient information for model fitting. 
+* Model fitting algorithm [details](https://openpharma.github.io/mmrm/main/articles/algorithm.html) used in `mmrm`.
+
+</details>
+<details><summary>Package Ecosystems Integration</summary>
+
+* Integration with [tidymodels](https://www.tidymodels.org/) package ecosystem
+  * Dedicated [parsnip](https://parsnip.tidymodels.org/) engine for linear regression
+  * Integration with [recipes](https://recipes.tidymodels.org/)
+
```

Comment:
As discussed we could also mention here the integration with the `tern` package ecosystem and link `tern.mmrm` in a sub-bullet, and maybe also the TLG catalog